### PR TITLE
Fixes #78 - Make tabs responsive on tools

### LINKED ISSUE #78: The Tools page layout is not responsive (mobile view)
The tabs on the Tools page are not responsive; i.e. they are extending outside of the container and breaking the layout.

### PR DESCRIPTION
**Pull request by danielvarcas**
- Changed tabs on Tools so they are responsive. Admittedly not as pretty as Semantic UI's menu tabs but they were breaking the layout on mobile view.
- Set Bootstrap theme primary colour to dark purple (same as yell box), so that the nav links are not bright blue.

The primary theme colour is a one-liner which can be changed easily if required. I did try a dark red to go with the Eden apple theme, but it looked a bit out of place with the current colour scheme.

(Note that the primary colour affects all link text and primary buttons, so a readable colour must be used.)

### CODE REVIEW

**Comment by danielvarcas** [comment]
Ok I had a bit of a horrible merge there and this branch isn't working currently. Will see if I can fix it and if not I may need to start again...

**Comment by danielvarcas** [comment]
I believe I've fixed the merge. Looks like our fork's develop branch was behind.

File: client/src/components/tools.jsx [hunk 1]
```diff
@@ -1,48 +1,57 @@
...
-    to={disabled ? undefined : to}
-    active={to === activeTab}
-    disabled={disabled}
-  >
-    {children}
-  </Menu.Item>
-);
+const TabItem = ({ to, activeTab, disabled = false, children }) => {
+  console.log(to);
+  console.log(activeTab);
```
Comment (InoUno) [inline client/src/components/tools.jsx:12]: Remove console logging

**Comment by slassen** [comment]
I wouldn’t be opposed to ideas other than tabs if you can think of something that looks nice

File: client/src/components/tools.jsx (hunk 1 above)
Comment (danielvarcas) [reply client/src/components/tools.jsx:12]: Woops, thanks.

**Comment by danielvarcas** [comment]
> Nice.
> One thing I noticed visually, is that it's pretty hard to see that they are actually tabs now, since they're white on a white background.

@InoUno I can override the styles so it looks like this:

image

I'm not sure how I feel about it yet though, maybe I just need to get used to it. Any thoughts?

This is the current default style for comparison:

image

**Comment by slassen** [comment]
IMO Bootstrap looks pretty dated anyways. I agree with Ino though, some clear borders would be ideal.

**Comment by danielvarcas** [comment]
@slassen Yeah, I might look at whether we can keep the Semantic UI styles but make them responsive. Or just make what I've got here prettier. It'll probably have to be at the weekend though. The Semantic UI styles themselves look good, it was just the responsiveness which has been as problem, but on the flip side I don't want to just make everything look bland. Will do some thinking this weekend.

If you're wanting to go live in the meantime, you can either keep it how it was but accepting the page is broken on mobile, or pull this in knowing the styles will change again. But if there's still time before going live then that's fine.

**Comment by slassen** [comment]
I’m ok going with Bootstrap. Just some borders would be good. Was just giving my thoughts overall about the dated comment.

**Comment by danielvarcas** [comment]
@slassen @InoUno So I added a class to the menu to make it response and it looks like this:

image

It's kind of annoying that the responsiveness isn't built in, but as we're only using this type of element on this one section of the site at the moment it's not going to cause any issues.

If you prefer this to the Bootstrap tabs I'll close this PR and open a new one with this change (to clarify: don't merge in this PR in that case).

**Comment by danielvarcas** [comment]
@slassen @InoUno I've created https://github.com/EdenServer/eden-web/pull/83 with the alternative fix. If you want to go ahead with that one, feel free to close this PR without merging. Thanks.

**Comment by slassen** [comment]
> > Nice.
> > One thing I noticed visually, is that it's pretty hard to see that they are actually tabs now, since they're white on a white background.
> 
> @InoUno I can override the styles so it looks like this:
> 
> image
> 
> I'm not sure how I feel about it yet though, maybe I just need to get used to it. Initially I think it looks a little dated. Any thoughts?
> 
> This is the current default style for comparison:
> 
> image

Are you able to push this change to this PR so I can compare both looks?

**Comment by danielvarcas** [comment]
@slassen Sure, I've committed a change which gives the nav tabs a visible outline.

**Comment by InoUno** [comment]
If going with the bootstrap solution, I think I would make the tabs stand out more. For example by adding more weight to the text, giving them a different color, adding icons, etc. 
I'd also make the active tab have the same background color as the content of the tab, to make them visually connected.

Example of more defined tab styling (although the bright blue may be too much for the current eden styling):
image

Also, if sticking with the new background around all of it (`bg-light` ), I'd remove the `<Segment>` part for each tool page, as there's too many layers in it now imo. The segment part would no longer be needed, since it would no longer detached from the tabs themselves.

**Comment by slassen** [comment]
Closing. Used your other solution instead: https://github.com/EdenServer/eden-web/pull/83

**Comment by danielvarcas** [comment]
@slassen good call, thanks :)

**Pull request closed without merging**